samples/bluetooth/mesh/prj.conf

```diff
@@ -5,9 +5,9 @@ CONFIG_BT_SETTINGS=y
 CONFIG_FLASH=y
 CONFIG_FLASH_PAGE_LAYOUT=y
```

Comment:
Can FLAG_PAGE_LAYOUT and FLASH_MAP follow from default? Are these required, and are they not default?

There seems to be a lot of configs that belong together that need to be copy pasted around to achieve persistent storage. It would be nice to reduce the list. We should be able to enable just one Kconfig option.